pyproject.toml

```diff
@@ -1,52 +1,57 @@
 [tool.poetry]
 name = "up42-py"
 version = "0.32.0"
-description = ""
-authors = ["Your Name <you@example.com>"]
+description = "Python SDK for UP42, the geospatial marketplace and developer platform."
+authors = ["UP42 GmbH"]
+license = "https://github.com/up42/up42-py/blob/master/LICENSE"
 readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
-requests =  "2.31.0"
-requests-oauthlib = "1.3.1"
-tenacity = "8.2.2"
-tqdm = "4.66.0"
-geojson = "3.0.1"
-geopandas = "0.13.2"
-pystac-client = "0.7.2"
+requests =  "^2.31.0"
```

Comment:
nitpick: how about we use `>=` everywhere?

Reply:
Honestly, I do not think that the issue is that we have to use the same everywhere as they do not serve the same purpose. 

>= (Greater Than or Equal To):
It's used to specify a minimum version requirement for a dependency.
For example, if you specify "requests >=2.0.0", it means your project requires at least version 2.0.0 of the "requests" library, but it can use any version that is greater or equal to that.

^ (Caret):
It's used to specify a compatible version range that allows for updates within a major version.
For instance, if you specify "requests ^2.0.0", it means your project can use any version within the 2.x range. It allows minor and patch version updates but keeps the major version fixed.
The main difference between the two is in how they handle version updates:

>= is more strict and only allows for patch and minor version updates.
^ is more permissive and allows for updates up to, but not including, the next major version.

For example, if the latest version of a package is 2.1.0:
"requests >=2.0.0" would accept any version from 2.0.0 to 2.1.0 but not 2.2.0.
"requests ^2.0.0" would accept any version from 2.0.0 to 3.0.0 but not 3.0.0 or higher.

So, you would use >= if you want to set a minimum version requirement and be more conservative about updates. You would use ^ if you want to allow for some flexibility and automatic updates within the same major version.

Unfortunately, in our case we need to figure out which dependencies can create conflicts and this can take some time, specially because when we deal with dependencies here we are dealing with not only pip but also conda releases. I think in the SDK case the dependencies are more complex and will need a separate effort to tackle, for Now I left the exact same that we have in the pip. Once we get rid of the requirement.txt files we can make sure all dependencies are working together.

WDYT.

Reply:
Thanks, man, not that I don't want to add it. I agree with you, but I think this dependency issue requires more careful consideration. Every time I touch the requirements, I am getting some error. And I not to mention when the time of release comes, we will have issues with the conda release that has a separate configuration for dependencies. I am still not sure which dependencies conda we will conflict. 

I really appreciate that you give detail review to this PRs. 

I am preparing myself for the release time. It will be nasty. :)